win32ss/include/ntuser.h

```diff
@@ -228,6 +228,7 @@ typedef struct _PROCMARKHEAD
 } PROCMARKHEAD, *PPROCMARKHEAD;
 
 #define UserHMGetHandle(obj) ((obj)->head.h)
```

Comment:
Maybe add a comment above to make users aware of the differences:
//use UserHMGetHandleSafe by default and if (obj) has not been checked for NULL already
//use UserHMGetHandle if (obj) has been checked for NULL already, or if (obj) is a function-call, to not call func twice